<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QDialog" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="skillsB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="professionPicLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>124</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="professionLabel">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>151</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="levelLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>186</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="skillPointsLabel">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>216</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="strLabel">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>126</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="intelLabel">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>155</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dexLabel">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>184</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="hpPB">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>425</y>
     <width>41</width>
     <height>176</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="mpPB">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>41</width>
     <height>176</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="bagLW">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>428</y>
     <width>265</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteB">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>650</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="goldLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="silverLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="copperLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inspectB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>650</y>
     <width>126</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useB">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>630</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="organizeBagB">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
